org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -504,7 +504,31 @@ GithubOidcSageBionetworksWebMonorepoInfra:
               "Resource": [ "arn:aws:s3:::prod.accounts.sagebionetworks.org",
                             "arn:aws:s3:::staging.accounts.sagebionetworks.org",
                             "arn:aws:s3:::prod.accounts.synapse.org",
-                            "arn:aws:s3:::staging.accounts.synapse.org"
+                            "arn:aws:s3:::staging.accounts.synapse.org",
+                            "arn:aws:s3:::prod-adknowledgeportalsynapse-org-websitebucket-1wcys549ufmd",
```

Comment:
would it be possible to wildcard some of these?  for example `arn:aws:s3:::*-adknowledgeportalsynapse-org-websitebucket-*`